en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
 gui.xaero_sort_reversed=Reversed Sort Order
 gui.xaero_sort_unsorted=Unsorted
 gui.xaero_sort_name=Name
-gui.xaero_sort_symbol=Initial
+gui.xaero_sort_symbol=Initials
 gui.xaero_sort_distance=Distance
-gui.xaero_box_cave_maps=Mapping mode that can go deeper than the surface blocks, mainly to display underground caves and interiors of buildings. The roof size stands for the size of a solid horizontal "square" of blocks that needs to be detected above you to activate the cave mode.
+gui.xaero_box_cave_maps=Mapping mode that can go deeper than the surface blocks, mainly to display underground caves and interiors of buildings. The ceiling size refers to the size of a horizontal "square" of solid blocks that has to be detected above you to activate the cave mode.
 gui.xaero_box_entity_amount=The limit per each entity category.
-gui.xaero_box_distance=Which ingame waypoints to show the distance to for.
-gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This does not override the "Distance To WP" option.
-gui.xaero_box_arrow_scale=Scale for the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_lock_player_heads=Display player heads instead of coloured dots even if the TAB key is not pressed.
-gui.xaero_box_cave_zoom=How much to zoom further in when the cave maps mode is active.
-gui.xaero_box_entity_depth=Display entities darker depending on their Y level relative to you.
-gui.xaero_box_height_limit=Maximum Y level distance for entities until they are no longer displayed.
-gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.,
-gui.xaero_toggle_map_waypoints=Toggle On-map Waypoints
-gui.xaero_entity_nametags=Entity Name Tags
+gui.xaero_box_distance=Specifies, for which in-game waypoints to show the distance to.
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=How much to zoom further in the minimap when the cave mode is active.
+gui.xaero_box_entity_depth=Display entities darker on the minimap depending on their Y level relative to you.
+gui.xaero_box_height_limit=Maximum Y-level distance for entities until they are no longer displayed on the minimap.
+gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.
+gui.xaero_toggle_map_waypoints=Toggle On-Map Waypoints
+gui.xaero_entity_nametags=Entities With Name Tags
```

Comment:
I'm pretty sure the original is more correct. It toggles name tags, not entities with name tags.

Reply:
When this option is turned on, the minimap displays entities **with** name tags, and when it is turned off, the minimap displays entities **without** name tags. I didn't think that it may cause confusion, to be honest.

Reply:
"Entities With Name Tags" sounds like it would completely remove the entities that have name tags, instead of just the name tags. Does it not?

Reply:
Probably you're right, although since that's my idea, it's not confusing to me. Do you think I should restore the previous version or perhaps you have a better wording in mind?

Reply:
Obsolete since version 20.24.0. Nothing more to say here.